ip/Zmods/ZmodAWGController/component.xml

```diff
@@ -2219,9 +2227,9 @@
       </xilinx:tags>
     </xilinx:coreExtensions>
     <xilinx:packagingInfo>
-      <xilinx:xilinxVersion>2021.1</xilinx:xilinxVersion>
+      <xilinx:xilinxVersion>2023.1.2</xilinx:xilinxVersion>
```

Comment:
Updating in a recent version may affect backward compatibility with previous versions. Please either confirm that the IP will still build in 2021.1 or reapply changes in 2021.1.